### PR TITLE
Content refresh

### PR DESCRIPTION
**Pull request by Vinnl**
This applies three primary changes across the entire website:
- An "alias" is now called a "mask"
- A "real email" is now called a "true email"
- We're using "subdomain" consistently now, rather than "domain"

Additionally, it adds a couple of strings that are only visible to screen readers, which we'll have better support for soon.

### CODE REVIEW

File: en/app.ftl
Comment (peiying2) [inline en/app.ftl:826]: `on an alias` -> `on a mask`

File: en/app.ftl
Comment (peiying2) [inline en/app.ftl:1058]: aliases -> masks, domain -> subdomain?

File: en/app.ftl
Comment (peiying2) [inline en/app.ftl:266]: should we remove this string as there is no replacement string for it?

**Comment by peiying2** [comment]
@Vinnl the terminology changes introduced in this PR meant that there are massive changes on the l10n side. We need to send a communication on the reasons behind these changes. It appears a few devs contributed to this PR. Some remove the deprecated strings while others mark those obsolete and provide new strings.  It feels like half of the strings from the product are updated. A few questions to address: 

- When do you plan to roll out the changes? Can we distinguish new strings from term changed strings in the rollout so we don't delay product development changes?
- What's the reason behind these term changes? Alias -> mask especially, needs some explanation. Can communities opt to not to change if `alias` works better in their language than `mask`, for example?
- What is the timeline to remove the `deprecated` strings?  Pros and cons on the deprecated strings:
  - If we remove them now, communities will be left what needs to be translated to work on, this is especially beneficial for those lagging behind. For those who already completed or close to it, all of sudden their product is half in English.
  - If we keep them both, new and deprecated, some  communities would be wasting their time working on strings no longer needed. Not everyone reads the comments, unfortunately.

File: en/app.ftl
Comment (maxxcrawford) [inline en/app.ftl:35]: (blocking) suggestion: We want to have a unified (non-numeric) variable name to identify all new strings from this update (instead of `-2`)

Can you adjust all new IDs to have the same appending name? (`-mask`)

```suggestion
meta-description-mask = { -brand-name-firefox-relay } makes it easy to create email masks that forward your messages to your true inbox. Use them to protect your online accounts from hackers and unwanted messages.
```

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
From our initial discussion with Chris, I thought the plan was to keep the variable name something non-numeric to make it easier to see the changes. 

Example that make this hard to parse: 
`faq-question-use-cases-answer-part1-2`, `faq-question-missing-emails-answer-a-2`

**Comment by tcinotto** [comment]
Hi @peiying2 Thank you! I'll try to answer the questions that I can. 

April 19th is our target release date, but if more time is needed for the community to localize the top languages (French and German) we can extend that target.

The reason to change the term from alias to mask is that we are looking to support and connect with more people and make privacy more accessible. Currently, our terminology in product is highly technical and we've found that there is a disconnect between many users and the content that we have been using. This is part of a wider effort to do a content refresh after we found various issues in user research. Additional user research was done on alias and masking and masking performed better compared to alias and got our message across and connected in a better way with users. 

In the future we might shift some of our design and branding towards mask. I understand though that that might not be easily localizable and might actually cause more confusion as the term might not have the exact same meaning across languages and cultures, so we'd like to try at least best effort to migrate to mask, but if not possible then I think it's okay to use alias. 

For the deprication of strings, I'll defer to @Vinnl

**Comment by Vinnl** [comment]
Apologies for springing this on you @peiying2, I thought you'd already been looped in on the changes before. I think Tony answered most questions, but as for

> Some remove the deprecated strings while others mark those obsolete and provide new strings.

I directly removed a couple of strings that already weren't in use anywhere in the codebase, so their removal should not affect anything. I marked most strings as deprecated, however, so that merging this in doesn't result in strings disappearing from the website until we actually make these changes go live (i.e. https://github.com/mozilla/fx-private-relay/pull/1710 - note how that is marked as "Do not merge").

And yeah, I realise we're asking a lot from our contributors. I know I'm at least personally very appreciative of all the fantastic work they're putting in :slightly_smiling_face: If there's anything we can do to make things easier for them, especially w.r.t. timing when to make these changes go live, I'm sure we're all happy to oblige.

Also:

> Can we distinguish new strings from term changed strings in the rollout so we don't delay product development changes?

So far, the new strings in this PR only cover existing functionality, adding accessible descriptions to them. That said, I took care to split add them in a separate commit (b4effb18216a4cdf9aa7036fdafce9d2f86dbe1e), so if you'd prefer that I split those out into a separate PR, it's easy to do so - just let me know.

I'll move on to your inline comments next :)

File: en/app.ftl
Comment (Vinnl) [reply en/app.ftl:35]: Ah, I didn't realise this was a hard requirement. Could you clarify on why this is so important as to be blocking? We're talking several more hours of _very_ tedious (and therefore error-prone) work, so I want to make sure that that's worth it.

To clarify, I did actually start with calling them `-mask`, but that made some IDs very confusing, since "mask" sounded as if it was part of the description. And of course, we'd get situations like `-2-mask`. Thus, I opted for consistency with our existing method of updating strings.

File: en/app.ftl
Comment (Vinnl) [reply en/app.ftl:611]: Ah, I should've removed that first one, since it was a new string and hadn't been localised yet. It's part of the new strings, so no longer in this PR, but I removed it in https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/69.

File: en/app.ftl
Comment (Vinnl) [reply en/app.ftl:797]: Nice catch: 83d970f4d6f4a7f709fe043269c72d10e866e097

File: en/app.ftl
Comment (Vinnl) [reply en/app.ftl:1008]: This one is marked as deprecated; those changes are applied in `forwarded-email-header-premium-banner-2`.

File: en/app.ftl
Comment (Vinnl) [reply en/app.ftl:263]: There is `premium-promo-hero-body-html-2`, though I now realise that it should end in `-html`, so I changed that to `premium-promo-hero-body-2-html` in 2da7a82ae1ace69792eaed94270ca7d8407bba7e.

**Comment by Vinnl** [comment]
Announcement here: https://discourse.mozilla.org/t/firefox-relay-a-major-update-for-the-next-release/95369

Merging :heavy_check_mark:

**Pull request merged**
